samples/Samples.Authorization/Samples.Authorization.csproj

```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-    <Nullable>enable</Nullable>
-    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
These lines are part of the template.  I would not remove them.  If someone copied the sample project, it would not work in their environment because the likely would not copy the Directory.Build.props file.

Reply:
All csprojs in repo already implicitly use `Directory.Build.props`/`Directory.Build.targets` from root. I reverted only using statements.

Reply:
These lines are part of the template. I would not remove them. If someone copied the sample project, it would not work in their environment because they likely would not copy the Directory.Build.props file.

Reply:
>  If someone copied the sample project

Sample project needs to be launched (preferably), not copied. I have never considered example projects as something that should be copied. Who knows what else can be forgotten when copying. There are a lot of options to miss something.

Reply:
Nevertheless we have countless issues where the sample code provided was copied directly from a sample, often erroneously.  These samples ARE designed to be the basis of a new project.  There is no reason why we should break this capability in this PR.

The more that users can copy and paste the samples into their own project, the better.  We constantly refer to project source/sample code within issues and instruct people to copy from there, as was done recently with the directives/validation sample.

Reply:
For example, within the JWT sample project, I included all the proper code for all this functionality:
- Use symmetric or asymmetric security keys
- Configure such keys from the app settings (which include app secrets or environment variables)
- Generate new symmetric or asymmetric security keys
- Configure audience and issuer constants
- Proper validation of JWT tokens over GET, POST and WebSocket connections
- Basic OAuth2 endpoint to authenticate against (meets spec)

This allows full use of the sample project as a _template_ for a new JWT-based GraphQL server, and includes all the necessary pieces except the mechanism to authenticate and/or store user credentials.  Compare this to other samples seen online, where the JWT sample code consists only of some extremely basic code that includes no signature validation at all, and the user is left to figure out all the complex details for themselves.

Providing fully-functioning _templates_ from which to provide a basis for a new application should be a valued part of this project as a whole.

Reply:
I expanded Samples section in readme.

Reply:
I disagree with those changes.

Reply:
So do you want me to revert all changes in every csproj file into samples folder, right?

Reply:
Yes absolutely. And the readme notes. If you want to discuss, do it in a separate PR.

Reply:
This PR was (originally) discussing adding analyzers, which I still disagree with. But if we disable all the pointless and unnecessary rules then we can merge that in - if you insist. I believe we have already discussed those changes. But changing the build props and csproj files is not within scope of this PR.

Reply:
I reverted all csprojs except one additional ignore in [Samples.Schemas.Chat.csproj](https://github.com/graphql-dotnet/server/pull/910/files#diff-f2ec7d4adc21fd8dada75dcc370233ccd772cac10afcb6b035c4aff2d0ccd09e)